python-backend/app/extensions.py

```diff
@@ -1,15 +1,22 @@
+import os
 from flask_caching import Cache
 from flask_jwt_oidc import JwtManager
 from flask_sqlalchemy import SQLAlchemy
 from flask_uploads import UploadSet, AllExcept, EXECUTABLES, SCRIPTS
 from flask_restplus import Api
+from pytz import timezone
```

Comment:
Did you end up using this? What's it for?

Reply:
No, I will remove it. I was trying to set the schedulers timezone but it wouldn't let me.